fitbenchmarking/fitting/minimizers.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Hard coding minimizer names in files within the fitbenchmarking package, unless there is a very good reason not to, should be avoided. This minimizer list may change with future versions of Mantid and hence a future user of fitbenchmarking would potentially have to change the guts of fitbenchmarking as I read this json file. Having such a file outside the fitbenchmarking package would be fine, e.g. in example_scripts

Reply:
I am trying to find a way such that the user can get a list of available minimizer names compatible with a certain algorithm and choose which minimizers they want/all minimizers. The implementation is not yet complete but the json file is a start.